src/js/stores/MesosStateStore.js

```diff
@@ -288,7 +297,12 @@ class MesosStateStore extends GetSetBaseStore {
     // Marathon tasks. This will give you a list of framework tasks including
     // the scheduler tasks or a list of Marathon application tasks.
     return frameworks.reduce(function(serviceTasks, framework) {
-      const { tasks = [], completed_tasks = {}, name } = framework;
+      const {
+        tasks = [],
+        completed_tasks = {},
```

Comment:
It's strange how this default value is an object, but above it's an array, and all other task data types are arrays.